packages/lib-classifier/src/plugins/tasks/DrawingTask/components/DrawingTask.js

```diff
@@ -29,30 +29,17 @@ const StyledText = styled(Text)`
     margin-top: 0;
   }
 `
-
-function storeMapper (stores) {
-  const { activeDrawingTool, setActiveDrawingTool } = stores.classifierStore.drawing
-  return {
-    activeDrawingTool,
-    setActiveDrawingTool
-  }
-}
-
-@inject(storeMapper)
 @observer
```

Comment:
I don't think this component needs to be an observer anymore. I tried removing it and looks like everything is still working as expected.

Reply:
I found that the task area didn't update when you created and deleted marks, but I can have another look in #1308.

Reply:
When I remove this, I see the point tool saying '0 drawn' after I've made two marks.